packages/core/src/App/Containers/Layout/footer.jsx

```diff
@@ -32,6 +34,11 @@ const Footer = ({
         <NetworkStatus />
         <ServerTime />
 
+        {landing_company_shortcode && (
```

Comment:
Question: Is this check necessary here? Is it possible to remove this check and rely on `.footer__links-extras:empty` styling?

Reply:
Thanks Mamali. I've made the changes.